examples/kubernetes/clustertemplates/main.tf

```diff
@@ -0,0 +1,5 @@
+data "vkcs_kubernetes_clustertemplates" "templates" {}
+
+output "available_templates_by_name" {
+  value = [ for template in data.vkcs_kubernetes_clustertemplates.templates.cluster_templates: {name = template.name, version = template.version}]
```

Comment:
could we format this adding some line breaks?

Reply:
fixed